drivers/crypto/crypto_tc_shim.c

```diff
@@ -25,6 +25,61 @@ LOG_MODULE_REGISTER(tinycrypt);
 
 static struct tc_shim_drv_state tc_driver_state[CRYPTO_MAX_SESSION];
 
+static int do_ecb_encrypt(struct cipher_ctx *ctx, struct cipher_pkt *op,
+			  u8_t *iv)
+{
+	struct tc_shim_drv_state *data =  ctx->drv_sessn_state;
+
+	if (op->in_len != 16) {
```

Comment:
Shouldn't you use `TC_AES_BLOCK_SIZE` instead of hardcoding 16 ?

Reply:
Ack